### ISSUE TITLE
Support importing QIF files

### ISSUE THREAD

**Issue by aclindsa** [aclindsa/moneygo#11]

**Comment by nirajbhatt7** [comment]
Parser seems to be simple. Here is someone who has done some work.

https://github.com/kalafut/qif

**Comment by aclindsa** [comment]
Yeah, I expect it won't be difficult, and that it'll just take a little time to figure out which QIF fields need to translate to which MoneyGo fields.
